sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 Xerox Corporation
+	1996-1999 Silicon Graphics
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
+	2007 NEC LE-IT
+	2016 Marek Vasut"
+LICENSE="MIT
+	GNU GPL v2"
```

Comment:
The [custom license][0] should also be included.

To add a custom license for a port, create a folder named "licenses" next to the recipe, and put the license text to a file named after the license. For this case, you would want to put the [license text][0] to `sys-libs/libatomic_ops/licenses/Boehm`

```diff
 LICENSE="MIT
+	Boehm
 	GNU GPL v2"
```

[0]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/doc/LICENSING.txt#L43